src/common/utils/utils.go

```diff
@@ -255,3 +255,8 @@ func IsContainIllegalChar(s string, illegalChar []string) bool {
 	}
 	return false
 }
+
+// JoinNumberConditions - To join number condition into string,used in sql query
```

Comment:
if it's just for sql query would it be more appropriate to move it to `dao` pkg?

Reply:
Updated